portfolio/src/main/webapp/script.js

```diff
@@ -25,7 +25,6 @@ async function getPhrase(){
             commentSection.appendChild(addMessage);  
             document.getElementById("section").appendChild(commentSection);
         }
-    }
```

Comment:
Make sure to unindent everything now that your for loop isn't nested in an if statement.